WatchApp Extension/Controllers/BolusInterfaceController.swift

```diff
@@ -86,7 +86,7 @@ final class BolusInterfaceController: WKInterfaceController, IdentifiableClass {
         var pickerValue = 0
 
         if let context = context as? WatchContext, let recommendedBolus = context.recommendedBolusDose {
-            pickerValue = pickerValueFromBolusValue(recommendedBolus * 0.75)
+            pickerValue = pickerValueFromBolusValue(recommendedBolus * 1.0)
```

Comment:
Why not remove the `* 1.0`?

Reply:
The red line is the line that will be edited out, the green line is the line that will be included after the PR is accepted.  The PR is to include the 1.0 multiplier in place of the 0.75 multiplier.